R/access-element.R

```diff
@@ -0,0 +1,55 @@
+#' @title Add Access Element
+#' @description The add access element creates a list that includes all the required
+#' elements for the access section of an EML document. Calling add_access() will output the 
+#' default public access permissions. To customize the access level provide values 
+#' for the allow_principal and add_permission arguments. 
+#' @param allow_principal The principal argument refers to who will have access
+#' to this dataset. The allow_principal argument takes in a string. 
+#' With no input the allow_principal will default to "public". To change
+#' to a different principal please input a new principal. 
+#' 
+#' The valid options for the principal are:
+#'    'public' - special principal indicate that any user or group has a particular access permission 
+#'     and other individual users or groups can be specified if they are defined in the 
+#'     "https://pasta.edirepository.org/authentication" authentication system 
+#' 
+#' @param allow_permission The permission argument refers to the type of access 
+#' that is allowed to the principal. The allow_permission argument takes in a string. 
+#' With no input the allow_permission will default to "read". This default permission
+#' allows the principal to read the dataset but not edit. To change to a different 
+#' permission please input a new permission. 
+#' 
+#' The valid options for permission are: 
+#'   'read' - allow viewing of the resource, 
+#'   'write' - allow modification of the resource (except for access rules),  
```

Comment:
same here, if you want to have it indented research how to have that formatting

Reply:
Added list formatting.